manage_arkime/cdk_interactions/cdk_client.py

```diff
@@ -9,7 +9,7 @@
 logger = logging.getLogger(__name__)
 
 def get_command_prefix(aws_profile: str = None, aws_region: str = None, context: Dict[str, str] = None) -> str:
-    prefix_sections = ["cdk"]
+    prefix_sections = ["node_modules/.bin/cdk"]
```

Comment:
Let's provide the absolute path to the `node_modules` directory rather than using a relative path.  You can assemble it pretty easily with the `get_repo_root_dir()` command [1].

[1] https://github.com/arkime/aws-aio/blob/main/manage_arkime/core/constants.py#L159